packages/byods/src/token-manager/index.ts

```diff
@@ -133,7 +176,7 @@ export default class TokenManager {
       throw new Error('orgId not provided');
     }
 
-    const serviceAppAuthorization = await this.getOrgServiceAppAuthorization(orgId);
+    const serviceAppAuthorization = await this.getTokenFromAdapter(orgId);
```

Comment:
In this method, we simply extract the token (from the private method) and refresh it regardless of validity, that is the reason I abstracted it out.
Open to any suggestions tho.